velox/common/encode/Base64.cpp

```diff
@@ -376,19 +386,20 @@ size_t Base64::calculateDecodedSize(const char* data, size_t& size) {
   return needed;
 }
 
+// static
 size_t Base64::decodeImpl(
     const char* src,
     size_t src_len,
     char* dst,
     size_t dst_len,
-    const ReverseIndex& reverse_lookup) {
+    const Base64::ReverseIndex& reverseIndex) {
   if (!src_len) {
     return 0;
   }
 
   auto needed = calculateDecodedSize(src, src_len);
   if (dst_len < needed) {
-    throw Base64Exception(
+    VELOX_USER_FAIL(
```

Comment:
Would be nice to make this API non-throwing: https://velox-lib.io/blog/optimize-try-more